diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/StringUtils.kt

```diff
@@ -63,3 +64,32 @@ fun String.removePrefix(): String {
     if (this.isUpperSnakeCase()) return this.substring(2)
     return this
 }
+
+fun String.editorDistance(another: String): Int {
```

Comment:
there is no existing libs for it? :))
if no - at least add Kdoc

Reply:
Guess I was just googling incorrectly. On the second attempt I found that it's implemented, for example, in Apache commons text. I'll try to use it then, wasn't happy about re-umplementing it either..

Reply:
then nevermind, that is absolutely ok

Reply:
It's even funnier - `kotlin-compiler-embeddable` contains `jline` inside of it, and Levenstein distance is implemented there.